src/test/java/com/epam/training/center/hw1/DataProvider/IsPositiveDataProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.training.center.hw1.DataProvider;
```

Comment:
pls, rename packages DataProvider, CalculatorTests according to convention (name must start from lowercase at least):
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html